packages/utils/src/cli-utils.ts

```diff
@@ -0,0 +1,13 @@
+import dotenv from 'dotenv';
+import { registerRequireHooks } from './';
+import path from 'path';
+
+export function cliInit(): void {
+    dotenv.config();
+    registerRequireHooks();
+}
+
+export const defaultMetaGlob = 'src/**/*.meta.ts?(x)';
+export function getDefaultWebpackConfigPath(projectPath: string): string {
```

Comment:
I'd rename this to `getProjectWebpackConfigPath()`, or simply export `'.autotools/webpack.config.js'`.